packages/dnspyex.vm/dnspyex.vm.nuspec

```diff
@@ -2,12 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>dnspyex.vm</id>
-    <version>6.2.0</version>
+    <version>6.2.0.20230201</version>
```

Comment:
should this be `6.3.0`?

Reply:
Good catch! This is because the package name is not the same as the dependency name (as some letters are capitalized in line 10). Should we modify line 10 to be `id="dnspyex"`?

Reply:
yes, or modify the update script?!

Reply:
Let's update the update script. I thought the last commit made the regex case insensitive?

Reply:
I'll update it. I've missed to update it for this case.